app/models/shopify/import/base.rb

```diff
@@ -0,0 +1,45 @@
+module Shopify
+  module Import
+    class Base
+      def initialize(credentials: {}, client: nil)
+        @client = client(credentials) unless client
+      end
+
+      def count(**opts)
```

Comment:
It's for `find_all` method, also we will need this wrapper later for update action.